tldr/cfg_extractor.py

```
@@ -2197,3 +2209,265 @@ def extract_elixir_cfg(source: str, function_name: str) -> CFGInfo:
+            false_block = self.new_block("body", else_block.start_point[0] + 1, else_block.end_point[0] + 1)
+            self.add_edge(branch_block_id, false_block.id, "false", f"not ({condition})")
+            self.current_block = false_block
+            self._visit_node(else_block)
+            if self.current_block and self.current_block.id not in self.exit_block_ids:
+                self.add_edge(self.current_block.id, after_if.id, "unconditional")
+        else:
+            self.add_edge(branch_block_id, after_if.id, "false", f"not ({condition})")
+
+        self.current_block = after_if
+
+    def _visit_while(self, node):
```

Ruff (0.14.14)

Missing return type annotation for private function `_visit_while`

Add return type annotation: `None`

(ANN202)